lib/OpenQA/Schema/Result/Assets.pm

```diff
@@ -198,6 +201,7 @@ sub limit_assets {
     if (opendir($dh, $OpenQA::Utils::assetdir . "/iso")) {
         my %isos;
         while (readdir($dh)) {
+            next if $_ =~ m/CURRENT/;
```

Comment:
I think `next if /CURRENT/;` is sufficient as `$_` is implied without right side.